docs/integrations/dlt.md

```diff
@@ -0,0 +1,87 @@
+# dlt
+
+SQLMesh enables efforless project generation using data ingested through [dlt](https://github.com/dlt-hub/dlt). This involves creating a baseline project scaffolding, generating incremental models to process the data from the pipeline's tables by inspecting its schema and configuring the gateway connection using the pipeline's credentials.
+
+## Getting started
+### Reading from a dlt pipeline
+
+To load data from a dlt pipeline into SQLMesh, ensure the dlt pipeline has been run or restored locally. Then simply execute the sqlmesh `init` command *within the dlt project root directory* using the `dlt` template option and specifying the pipeline's name with the `dlt-pipeline` option:
+
+```bash
+$ sqlmesh init -t dlt --dlt-pipeline <pipeline-name> dialect 
```

Comment:
i think you'll probably also want to define a start date or no?

Reply:
It is generated automatically

Reply:
Yes also added a function to extract the start date from the pipeline directly to be set in the sqlmesh `config.yaml`